src/Features/CSharpTest/Wrapping/ParameterWrappingTests.cs

```diff
@@ -947,6 +947,34 @@ public C(int i,
             }
             """);
 
+    [Fact]
+    public Task TestInPrimaryConstructor()
+        => TestInRegularAndScriptAsync(
+            """
+            class [||]C(int i, int j) {
+            }
+            """,
+            """
+            class C(int i,
+                    int j) {
+            }
+            """);
+
+    [Fact]
+    public Task TestInPrimaryConstructorBase()
+        => TestInRegularAndScriptAsync(
+            """
+            class C(int i, int j)
+                : [||]Base(i, j) {
+            }
+            """,
+            """
+            class C(int i, int j)
+                : Base(i,
+                       j) {
+            }
+            """);
```

Comment:
fwiw, these are arguments, and should be ArgumentWrappingTests as these are arugments.

Also, we have a TestAllWrappingCasesAsync that you should use.

see the tests there for a smattering of cases to consider.  Basically, where it's not wrapped, or already wrapped, and all the different wrapping forms.

like TestTwoParamWrappingCases and Test_AllOptions_NoInitialMatches and Test_LongWrapping_VariadicLengthIds